packages/amplify-category-auth/src/provider-utils/awscloudformation/utils/generate-auth-trigger-template.ts

```diff
@@ -91,32 +115,66 @@ export const generateNestedAuthTriggerTemplate = async (
   }
 };
 
-export const createCustomResourceForAuthTrigger = (authTriggerConnections: AuthTriggerConnection[]): Record<string, unknown> => {
-  const stack = new CustomResourceAuthStack(undefined as unknown as Construct, 'Amplify', {
-    description: 'Custom Resource stack for Auth Trigger created using Amplify CLI',
-    authTriggerConnections,
-  });
-  const cfn = stack.toCloudFormation();
-  return cfn;
+/**
+ * creates custom resource for cognito triggers
+ */
+// eslint-disable-next-line spellcheck/spell-checker
+export const createCustomResourceforAuthTrigger = async (
```

Comment:
This looks like a rebasing/merge issue, but `createCustomResourceForAuthTrigger` has correct camel casing. Can that be used instead? The eslint rule won't need to be disabled then.